### ISSUE TITLE
Improve overall Accessibility on Storefront Platform

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2118]
Scope
---
The Sorefront of the Design System should be seen as "best practice" showing the possibilities of an accessible website. Therefore we should implement a better UX with the following quick fixes and optimizations to support SPA handling.

Acceptance criteria
---
- [ ] Storefront should met all accessibity criteria

Subtasks
---
- [ ] Add page specific page titles
- [ ] Add support for SPA route change (focus `h1` after loading the content)
- [ ] Add skip to content link
- [ ] Add current state of navigation item
- [ ] Optimize Search
- [ ] Change focus outline of native selects to PDS style
